src/frontend/data/text-data/components/check-balance/step-2/check-wallet.ts

```diff
@@ -2,7 +2,7 @@ export const checkWalletContent = {
     heading: "Check your wallet's balance",
     paragraph: "You can check you wallet's balance by entering your address.",
     input: {
-        name: "walletAddress",
+        name: "targetAddress",
```

Comment:
Would have though "wallet" should be used here, target seems to vague to me.

Reply:
Hmmm, every address is a wallet's address. I wanted to make sure future readers of this code understand that this is where tokens are sent to, so I used the word target. I think it's clearer this way, because in our system money always comes FROM a user's wallet and tokens are sent TO a user's wallet.

Reply:
Ok that's clear thx